packages/libs/error-stack/src/macros.rs

```diff
@@ -279,7 +280,11 @@ mod tests {
             "additional message",
             "Context A"
         ]);
-        // TODO: check `err.frames().next().kind() == FrameKind::Context`
+        assert_eq!(kinds(&err), [
```

Comment:
Can we also test one or two with multiple contexts and attachments, just to make sure?

Reply:
Added a new test:
```rust
#[test]
fn kinds() {
    use FrameKind::{Attachment, Context};

    let report = Report::new(ContextA);
    let report = report.attach("A1");
    let report = report.attach("A2");
    let report = report.change_context(ContextB);
    let report = report.attach("B1");
    let report = report.attach("B2");

    assert_eq!(frame_kinds(&report), [
        Attachment, Attachment, Context, Attachment, Attachment, Context
    ]);
    assert_eq!(messages(&report), [
        "B2",
        "B1",
        "Context B",
        "A2",
        "A1",
        "Context A"
    ]);
}
```